fec/management/commands/createsu.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+import os
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        if not User.objects.filter(username="admin").exists():
+            User.objects.create_superuser("cgarretty", "admin@admin.com",
+                                          os.environ['SUPERUSER_KEY'])
```

Comment:
Nice use of the environment!

Could you add `SUPERUSER_KEY=password` or something to the `.env.example` file, to remind us this exists? I like having every possible environment variable listed in there, it helps when debugging a production issue to know what variables _should_ exist